app/views/layouts/application.html.erb

```diff
@@ -6,14 +6,26 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render partial: "shared/cdn_assets" %>
```

Comment:
Great job adding your cdn assets to a partial! This helps keep the layout file clean